pkg/platform/types.go

```diff
@@ -36,10 +36,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-//
-// Auth
-//
+type ProxyLogsSource string
 
+const (
+	ProxyLogsSourceES  ProxyLogsSource = "elasticsearch"
+	ProxyLogsSourceK8s ProxyLogsSource = "kubernetes"
+)
+
+// Auth
```

Comment:
why change the `//` formatting?

Reply:
@TomerShor actually that wasn't me, probably IDE has changed it somehow, i'll revert